### PR TITLE
Remove self-referencing links for webhooks docs

### PR DESCRIPTION
**Pull request by annegentle**
Fix for Issue #4282; docs-only change.

### CODE REVIEW

File: docs/webhooks.rst [hunk 1]
```diff
@@ -26,10 +29,14 @@ project was not imported through a connected account, you may need to
 manually configure a webhook for your project.
 
 To manually set up a webhook, click **Add integration** on your project's
-**Integrations** admin dashboard page and select the integration type you'd like
+**Integrations** Admin dashboard page and select the integration type you'd like
 to add. After you have added the integration, you'll see a URL for the
-integration on the :ref:`integration detail page <webhooks:Webhook Integrations>`. Use this
-URL when setting up a new webhook with your provider -- these steps vary
+integration once you go to readthedocs.org/dashboard, log in, and then go to
```
Comment (stsewd) [inline docs/webhooks.rst]: I'm not sure if is the best to point to the users to that link and the log in step, I think if the users are reading this they already in their project's admin page (the above step already point to that).

File: docs/webhooks.rst [hunk 2]
```diff
@@ -26,10 +29,14 @@ project was not imported through a connected account, you may need to
...
 To manually set up a webhook, click **Add integration** on your project's
-**Integrations** admin dashboard page and select the integration type you'd like
+**Integrations** Admin dashboard page and select the integration type you'd like
 to add. After you have added the integration, you'll see a URL for the
-integration on the :ref:`integration detail page <webhooks:Webhook Integrations>`. Use this
-URL when setting up a new webhook with your provider -- these steps vary
+integration once you go to readthedocs.org/dashboard, log in, and then go to 
+**Admin** > **Integrations** to see or create an integration.
+
+As an example, the URL pattern looks like this: *readthedocs.org/api/v2/webhook/<project-name>/<nnnnnn>/*.
```
Comment (stsewd) [inline docs/webhooks.rst]: We can use `<id>` instead of `<nnnn>`

File: docs/webhooks.rst [hunk 3]
```diff
@@ -71,8 +82,8 @@ Using the generic API integration
 
 For repositories that are not hosted with a supported provider, we also offer a
 generic API endpoint for triggering project builds. Similar to webhook
-integrations, this integration has a specific URL, found on the
-:ref:`integration detail page <webhooks:Webhook Integrations>`.
+integrations, this integration has a specific URL, found on the project's
+**Integrations** Admin dashboard page on readthedocs.org.
```
Comment (stsewd) [inline docs/webhooks.rst]: Also here, probably the users already know that the project is in readthedocs.org, right?

File: docs/webhooks.rst [hunk 4]
```diff
@@ -90,8 +101,8 @@ branches
     Default: **latest**
 
 token
-    The integration token. You'll find this value on the
-    :ref:`integration detail page <webhooks:Webhook Integrations>` page.
+    The integration token. You'll find this value on the on your project's
```
Comment (stsewd) [inline docs/webhooks.rst]: `on your` shouldn't be there

File: docs/webhooks.rst (hunk 1 above)
Comment (annegentle) [reply docs/webhooks.rst]: Agreed, good point. Revising.

File: docs/webhooks.rst (hunk 2 above)
Comment (annegentle) [reply docs/webhooks.rst]: Thanks - and is that formatting okay with angle brackets and italic?

File: docs/webhooks.rst (hunk 3 above)
Comment (annegentle) [reply docs/webhooks.rst]: Yeah, I'm revising in the next patch, see what you think... I still find it a bit disorienting if I'm not in that Dashboard very often.

File: docs/webhooks.rst (hunk 2 above)
Comment (stsewd) [reply docs/webhooks.rst]: Yeah, I think is fine

File: docs/webhooks.rst (hunk 3 above)
Comment (stsewd) [reply docs/webhooks.rst]: I think it is great now, we can change the wording here too. Thanks for the contribution btw ;)

**Comment by ericholscher** [comment]
Looks good. Thanks for the PR!

**Pull request merged**